libs/gdk/src/lib/gdk-import/gdk-import.component.html

```diff
@@ -0,0 +1,31 @@
+<pre>
```

Comment:
why `gdk-import` wasn't done in another PR?

Reply:
I've create this component when I needed to manipulate html code to display the "fun area" widget. So, instead to waste time in my own component, I created another component that can be used for other person. If I tried to create in separated pull request, I needed to refactor an existent component (of shell folder). Because those components doesn't use ReactiveForms (that is requested by gdk-import).